src/components/cc-addon-option-form/cc-addon-option-form.js

```diff
@@ -128,6 +130,17 @@ export class CcAddonOptionForm extends LitElement {
 
           --cc-icon-size: 1.25em;
         }
+
+        /* Temporary workaround for cc-addon-elasticsearch-options because the skeleton parts should only be the <strong> elements but they are nested inside i18n strings */
+        .option-warning .skeleton {
+          color: var(--cc-color-text-weak);
+        }
+
+        /* Temporary workaround for cc-addon-elasticsearch-options because the skeleton parts should only be the <strong> elements but they are nested inside i18n strings */
```

Comment:
suggestion: maybe add a `TODO` or `FIXME` on the comment for later

Reply:
good idea, I'll do that!